controllers/provider-azure/pkg/controller/worker/machines.go

```diff
@@ -174,7 +186,7 @@ func (w *workerDelegate) generateMachineConfig(ctx context.Context) error {
 					},
 					"machineType":  pool.MachineType,
 					"image":        image,
-					"volumeSize":   volumeSize,
+					"osDisk":       osDisk,
```

Comment:
Doing this will trigger a rolling update of all machines of all clusters (in their maintenance windows, if configured). Can you please add this to the release notes?

Reply:
Done.